pkg/sql/catalog/lease/lease.go

```diff
@@ -821,7 +821,7 @@ func purgeOldVersions(
 	// if we hit this case. Note: This scenario is impossible to hit in the real
 	// world since the lease duration is never set to 0.
 	desc, _, err := t.findForTimestamp(ctx, m.storage.clock.Now())
-	if isInactive := catalog.HasInactiveDescriptorError(err); err == nil || isInactive || errors.Is(err, errRenewLease) {
+	if isInactive := catalog.HasInactiveDescriptorError(err); err == nil || isInactive {
```

Comment:
Can you try reverting this part of the change and check if the test is stable? I think it's causing CI failures

Reply:
Thanks Faizan! I removed the comment.
<!-- Reviewable comment -OEVeVx0-T_aERcC_NnX:bj9lydb -->
<!-- Sent from Reviewable.io -->